server/src/main/java/org/apache/iotdb/db/engine/merge/task/MergeTask.java

```diff
@@ -111,6 +111,12 @@ private void abort() throws IOException {
   }
 
   private void doMerge() throws IOException, MetadataException {
+    if (resource.getSeqFiles().size() == 0) {
+      logger.info("{} starts to merge {} seqFiles, so will abort task.", taskName ,
```

Comment:
Maybe log like "No sequence file to merge into, abort task" can describe the reason of abort more specifically.

Reply:
Thanks @SilverNarcissus , I have alter log.. please review...